db/README.md

```diff
@@ -1,10 +1,42 @@
-database for the enermaps2 project.
+# Database for the enermaps project
 
 We use postgresql with postgis. The initial database schema 
 can be found in db/add_dataset_db.sql
 
+The sql schema will be loaded only if the `db-data` volume is not already present.
+You can use `docker-compose down --volumes` and `docker volume rm db-data` before running the app to load the schema and start from an empty db.
```

Comment:
This information is already in the README at the root of the directory, I see two solution there:
* Either you have a documentation specific to the db that drop only the db volume
* Or you rely on the README at the root of the repository and add a reference to it

Reply:
No, it's not, at least the `docker volume rm db-data` part: it took me a while to understand that rebuilding the db was not enough.  Anyway, I don't see a major problem in repeating some concepts. But indeed we could avoid having a separate readme for each service and group all doc in the main readme.

Reply:
even in this section: https://github.com/enermaps/enermaps#cleanup ? `docker-compose down -v` seems to be pretty similar to the command you proposed, on my machine it prints the following:

```sh
$ docker-compose down -v                                                                                 
Removing network enermaps2_default
Removing volume enermaps2_upload-data
Removing volume enermaps2_db-data
Removing volume enermaps2_data
```

Is there something I missed ?

Reply:
Ah I see your point. For some reason, in my machine it doesn't remove the `db-data` volume...
```
$  docker-compose down -v
Stopping enermaps_api_1              ... done
Stopping enermaps_frontend_1         ... done
Stopping enermaps_data-integration_1 ... done
Stopping enermaps_db_1               ... done
WARNING: Found orphan containers (enermaps_cm-countclass_1, enermaps_thumbnails_1) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.
Removing enermaps_api_1              ... done
Removing enermaps_cm-multiply_1      ... done
Removing enermaps_flower_1           ... done
Removing enermaps_redis_1            ... done
Removing enermaps_cm-geofile_1       ... done
Removing enermaps_frontend_1         ... done
Removing enermaps_data-integration_1 ... done
Removing enermaps_db_1               ... done
Removing network enermaps_default
Removing volume enermaps_upload-data
ERROR: remove enermaps_upload-data: volume is in use - [9c24dbd20c779b98360e1e8da7d4b66fd978821f6a4594419d4795a7d75f6904]
```

Reply:
Sounds like another problem, see:
```
WARNING: Found orphan containers (enermaps_cm-countclass_1, enermaps_thumbnails_1) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.
```
it means that you have countainers that are still running in this project that are not currently in the docker-compose file. You can check for those and kill them using:

```sh
docker ps | grep -v CONTAINER | awk '{ print $1  }' | xargs docker stop
```

Reply:
The command is still there, do you want to remove it as it cannot be made specific to this container ?